README.md

```diff
@@ -100,7 +106,7 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
-Replace this with your answer
+A schema is a logical structure and grouping of information in a database and usually represented as objects and/or tables. For people and wishes, we would say that a person has many wishes and that the wishes belong to one person.
 ```
```

Comment:
not really represented as objects. Just tables. It uses tables to generate rows that are modeled after objects.